src/routers/v1/SearchRouter.js

```diff
@@ -5,13 +5,15 @@ import type Request from 'express';
 import AllStopUtils from '../../utils/AllStopUtils';
 import ApplicationRouter from '../../appdev/ApplicationRouter';
 import RequestUtils from '../../utils/RequestUtils';
+import Constants from '../../utils/Constants';
 
 const BUS_STOP = 'busStop';
 const cacheOptions = {
   max: 10000, // Maximum size of cache
   maxAge: 1000 * 60 * 60 * 24 * 5, // Maximum age in milliseconds
 };
 const cache = LRU(cacheOptions);
+const placeIDToCoordsCache = LRU(cacheOptions);
```

Comment:
should this cache have a `maxAge`, or can we just store placeIDs indefinitely?

Reply:
the placeID to coordinate shouldn't change so i dont need we need a maxAge